src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -21,8 +21,12 @@
 from apps.core.decorators import log_request
 from apps.modeling import geoprocessing
 from apps.modeling.mapshed.calcs import streams
-from apps.modeling.mapshed.tasks import nlcd_streams
+from apps.modeling.mapshed.tasks import (collect_data,
+                                         convert_data,
+                                         multi_mapshed,
+                                         nlcd_streams)
 from apps.modeling.serializers import AoiSerializer
+from apps.modeling.views import _parse_input as _parse_modeling_input
```

Comment:
Importing from `modeling.views` is a little messy here. I expect that this will be removed once everything is moved to `geoprocessing_api`.